lang/en/block_opencast.php

```diff
@@ -359,6 +359,8 @@
 $string['expirydate'] = 'Expiry Date';
 $string['failedtogetvideo'] = 'Cannot get event data from Opencast.';
 $string['failedtransferattempts'] = 'Failed transfer attempts: {$a}';
+$string['faileduploadretrylimit'] = 'Limit failed upload attempts';
+$string['faileduploadretrylimitdesc'] = 'Every upload job will be archived after the failed attempts limit is reached. When an upload job is archived, teachers have to decide whether to delete the job or to manually put it back into the queue. <br/> If the event status notification "eventstatusnotificationenabled" is enabled, teachers will get informed about archived status as well. <br/><strong>NOTE:</strong> An empty or 0 (default) value is considered as no limit.';
```

Comment:
Maybe write that the uploader will always be informed, and the teachers as well, if the setting is enabled